webview-ui/src/i18n/locales/en/chat.json

```diff
@@ -45,6 +45,16 @@
 		"title": "Approve",
 		"tooltip": "Approve this action"
 	},
+	"read-batch": {
+		"approve": {
+			"title": "Approve All"
+		}
+	},
+	"read-reject": {
+		"approve": {
```

Comment:
This key structure is weird. Seems like it should be under `read-batch.deny.title`